hooks/useAuthentication.js

```diff
@@ -55,15 +55,8 @@ function hasPendingDeletion(userData) {
   return userData?.requestDeletion?.[0] === "requested";
 }
 
-function normalizeLanguage(language) {
-  if (language !== "en" && language !== "es") return null;
-  if (language === "en") return "en-US";
-  return language;
-}
-
 // TODO: store refresh token in cookie so token can be refreshed after browser session ends
-export default function useAuthentication(data) {
-  const { push } = useRouter();
+export default function useAuthentication() {
```

Comment:
removed most of the redirecting to preferred language stuff, our routing handles this